src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -393,6 +398,7 @@
   (let [handler (RequestLogHandler.)
         logger (RequestLogImpl.)]
     (.setFileName logger (:access-log-config config))
+    (.setQuiet logger true)
```

Comment:
Nice! I couldn't figure out how to suppress this logging in the Access Logging tests when I added them. Glad to see these log messages are going away.